test/navigation/NavigatorSpec.scala

```diff
@@ -133,6 +133,12 @@ class NavigatorSpec extends SpecBase {
           routes.WhatIsGIINController.onPageLoad(NormalMode)
       }
 
+      "must go from HaveGIIN to WhatIsGIIN when user answer is no" in {
```

Comment:
The test goes from HaveGIIN to WhereIsFIBased. Does the description require updating?

Reply:
Oops mistyped that - on it 👍